tests/phpunit/api/v3/AddressTest.php

```diff
@@ -146,7 +146,25 @@ public function testCreateAddressWithMasterRelationshipOrganization() {
     $this->callAPISuccess('relationship', 'getcount', array(
       'contact_id_a' => $individualID,
       'contact_id_b' => $this->_contactID,
-    ));
+    ), 1);
```

Comment:
The existing unit test of this functionality was actually not testing anything because no expected value was set here.